code/controllers/configuration/configuration.dm

```diff
@@ -231,7 +231,13 @@
 	return E.ValidateAndSet("[new_val]")
 
 /datum/controller/configuration/proc/LoadMOTD()
-	motd = file2text("[directory]/motd.txt")
+	var/list/motd_contents = list()
+
+	for (var/motd_file in CONFIG_GET(str_list/motd))
```

Comment:
if the config is empty, you should check for motd.txt and if it exists, use it.

config additions should do their best to preserve old behavior